src/components/cardbuilder/roundcard.css

```diff
@@ -1,7 +0,0 @@
-.card-round:focus > .cardBox-focustransform {
-    transform: scale(1.26, 1.26);
```

Comment:
Isn't this used be people in the item details page?

Reply:
I didn't find any reference to this file, making a search in the repository i can't find any reference. Can you confirm it?

Reply:
right [here](https://github.com/jellyfin/jellyfin-web/blob/master/src/scripts/site.js#L787)

Reply:
Right, but that module isn't used anywhere, so that's why I removed it. But I didn't checked on the plugins so it could be used there